emrt/necd/content/browser/statisticsview.py

```diff
@@ -1,26 +1,23 @@
+from datetime import datetime
 from Products.CMFCore.utils import getToolByName
-from emrt.necd.content.reviewfolder import IReviewFolder
-from five import grok
+from Products.Five import BrowserView
 from zope.component import getUtility
 from zope.schema.interfaces import IVocabularyFactory
 
-import itertools
 import copy
+import itertools
 import operator
-
 import tablib
-from datetime import datetime
 
-grok.templatedir('templates')
 
+class StatisticsView(BrowserView):
 
-class StatisticsView(grok.View):
-    grok.context(IReviewFolder)
-    grok.name('statistics')
-    grok.require('cmf.ManagePortal')
+    def __init__(self, context, request):
```

Comment:
Don't do intensive tasks on `__init__` as `getAdaptersFor` (`BrowserView`s are multi-adapters) will call `__init__` every time. Instead move those to `__call__`.